Cargo.toml

```diff
@@ -49,21 +49,38 @@ r2d2 = { version = "0.8.8", optional = true }
 crc16 = { version = "0.4", optional = true }
 rand = { version = "0.8", optional = true }
 # Only needed for async_std support
-async-std = { version = "= 1.8.0", optional = true}
+async-std = { version = "= 1.8.0", optional = true }
 async-trait = { version = "0.1.24", optional = true }
 
 # Only needed for TLS
 native-tls = { version = "0.2", optional = true }
 tokio-native-tls = { version = "0.3", optional = true }
 async-native-tls = { version = "0.4", optional = true }
 
+# Only needed for JSON Support
+serde = { version = "1.0.137", optional = true }
+serde_json = { version = "1.0.81", optional = true }
+
 # Optional aHash support
 ahash = { version = "0.7.6", optional = true }
 
 [features]
 default = ["acl", "streams", "geospatial", "script"]
+json = []
```

Comment:
added new `json` feature to make sure the new stuff doesn't go where it doesn't need to go